tests/test_connections/test_ledger/test_ledger.py

```diff
@@ -82,14 +81,6 @@ async def test_not_hanging(self) -> None:
         ):
             await ledger_connection.send(dummy_envelope)
 
-        # call receive
-        with mock.patch.object(
-            LedgerConnection,
-            "receive",
-            return_value=dummy_receive(ledger_connection.receiving_tasks),
-        ):
-            receive_task = asyncio.ensure_future(ledger_connection.receive())
-
```

Comment:
the ledger_connection.receive() already has the issue, why not call it directly?